virttest/utils_windows/virtio_win.py

```diff
@@ -132,26 +132,32 @@ def _get_netkvmco_path(session):
         raise exceptions.TestError(err)
 
     middle_path = "%s\\%s" % (guest_name, guest_arch)
-    find_cmd = 'dir /b /s %s\\netkvmco.dll | findstr "\\%s\\\\"'
-    find_cmd %= (viowin_ltr,  middle_path)
-    netkvmco_path = session.cmd(find_cmd).strip()
-    LOG.info("Found netkvmco.dll file at %s" % netkvmco_path)
-    return netkvmco_path
-
+    for file_name in ["netkvmco.dll", "netkvmco.exe"]:
+        find_cmd = 'dir /b /s "%s" | findstr "%s" | findstr "%s"' \
+                               % (viowin_ltr, middle_path, file_name)
+        status, output = session.cmd_status_output(find_cmd)
+        if status != 0:
+            continue
+        netkvmco_path = output.strip().split("\n")[0]
+        LOG.info("Found %s file at %s" % (file_type, netkvmco_path))
```

Comment:
There is an undefined parameter `file_type` here.
It will introduce an error like that: https://github.com/avocado-framework/avocado-vt/actions/runs/7335520040/job/19973542039?pr=3808

Reply:
Sorry for that. 
Since I have run the code many times before, I only focused on the code part this time and I did not notice the variables inside the `LOG.info`.  Could you help me to change it together in your patch? Thanks in advance : )

Reply:
It's ok for me.I can help to modify the code to the following, would you please help confirm if it is your expected.
`LOG.info("Found file at %s" % netkvmco_path)`

Reply:
@leidwang Thanks for the heads up. @heywji Please sent an independent hotfix for it.

Reply:
> @leidwang Thanks for the heads up. @heywji Please sent an independent hotfix for it.

CC @nickzhq